### ISSUE TITLE
When called for an image component, setColor does not update image color. 

### ISSUE THREAD

**Issue by dvbridges** [psychopy/psychojs#29]
In PsychoPy, you can change the color of your image component on every repeat or every frame using the ```Color``` parameter in the Builder image component. However, the code that is used for the JavaScript is not working - where "image" is the name of the image component:
```
image.setColor(new util.Color('red'))
```

**Comment by peircej** [comment]
The implementation in psychopy is essentially to multiply the image pixel color by the RGB of the designated color. We now do this with a shader but it used to be using textures combined with glColor

**Comment by wakecarter** [comment]
Have you checked that this isn't related to writing 'red' instead of new util.Color([1, -1, -1]); ?

**Comment by dvbridges** [comment]
Yes it fails both ways. For named colors, you can use any of those listed as a named color in PsychoJS

https://github.com/psychopy/psychojs/blob/8276bd790a682ab276052946f66cb4d591ad0eb6/js/util/Color.js#L493

**Comment by laszlotalas** [comment]
Hi, I'm wondering if this has been resolved or anyone found a way around it. I'm trying to recolour an empty white image by setting a new colour. Works brilliantly locally, fails online. Unfortunately using a Polygon is not an option as I need a mask on the image.
